force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +91,26 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
-        // Create a set to store the unique words
-
-        // Add each word to the set
-
+        List<String> wordList = sentence.split(' ');
+        System.debug(wordList);
+        Set<String> wordSet = new Set<String>();// Create a set to store the unique words
+        String finalSentence = '';
+        for(String words : wordList){ // Add each word to the set
+            if(wordSet.contains(words)){
+                continue;
+            }
+            wordSet.add(words);
```

Comment:
You can construct a Set or List from another Set or List.
Look at String.join to combine a collection of strings into a CSV